mkosi.conf.d/20-kali/mkosi.conf

```diff
@@ -0,0 +1,11 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+[Match]
+Distribution=kali
+
+[Distribution]
+Repositories=non-free-firmware
+
```

Comment:
Is this an actual repository for Kali or did you just copy paste this?

Reply:
As Kali follows Debian (testing) closely, it is a real and required repository (for microcode packages).